File: content/api-documentation/how-to/account/examples/gainloss.go
package main

import (
	"fmt"
	"log"

	"github.com/alpacahq/alpaca-trade-api-go/alpaca"
)

func main() {
	alpaca.SetBaseUrl("https://paper-api.alpaca.markets")

	// Get account information.
	account, err := alpaca.GetAccount()
	if err != nil {
		log.Fatalln(err)
	}

	// Calculate the difference between current balance and balance at the last market close.
	balanceChange := account.Equity.Sub(account.LastEquity)

	fmt.Println("Today's portfolio balance change:", balanceChange)
}

File: content/api-documentation/how-to/account/examples/gainloss.js
const Alpaca = require('@alpacahq/alpaca-trade-api')
const alpaca = new Alpaca()

// Get account information.
alpaca.getAccount().then((account) => {
	// Calculate the difference between current balance and balance at the last market close.
    const balanceChange = account.equity - account.last_equity

    console.log('Today\'s portfolio balance change:', balanceChange)
})
